src/Bindings/ManualBindings_BlockArea.cpp

```diff
@@ -951,6 +951,105 @@ static int SetRelBlock(lua_State * a_LuaState)
 
 
 
+static int tolua_cBlockArea_SetBlockTypeMeta(lua_State * a_LuaState)
+{
+	// Check the common params:
+	cLuaState L(a_LuaState);
+	if (!L.CheckParamSelf("cBlockArea"))
+	{
+		return 0;
+	}
+
+	// Read the common params:
+	cBlockArea * self;
+	if (!L.GetStackValues(1, self))
+	{
+		return cManualBindings::ApiParamError(a_LuaState, "Cannot read the 'self' param.");
+	}
+
+	// Check if block types and metas are present:
+	if (!self->HasBlockTypes() || !self->HasBlockMetas())
+	{
+		return cManualBindings::ApiParamError(a_LuaState, "The area doesn't contain the datatypes (%d) and (%d).", cBlockArea::baTypes, cBlockArea::baMetas);
```

Comment:
Why not just say that the area doesn't contain baTypes and baMetas? That's more understandable for plugin authors.